Fix warning about unused variable

```
/home/dustin/p/taskwarrior/src/commands/CmdEdit.cpp:624:7: warning: variable 'ignored' set but not used [-Wunused-but-set-variable]
  624 |   int ignored = chdir(location._data.c_str());
      |       ^
1 warning generated.
```
I suppose this is a new clang warning? I assume `ignored` was a way to intentionally ignore this value, but there's probably a better way to explain that to the compiler.

--- a/src/commands/CmdEdit.cpp
+++ b/src/commands/CmdEdit.cpp
@@ -619,10 +619,9 @@ CmdEdit::editResult CmdEdit::editFile(Task& task) {
   auto dateformat = Context::getContext().config.get("dateformat.edit");
   if (dateformat == "") dateformat = Context::getContext().config.get("dateformat");
 
-  // Change directory for the editor
+  // Change directory for the editor, doing nothing on error.
   auto current_dir = Directory::cwd();
-  int ignored = chdir(location._data.c_str());
-  ++ignored;  // Keep compiler quiet.
+  chdir(location._data.c_str());
 
   // Check if the file already exists, if so, bail out
   Path filepath = Path(file.str());
@@ -702,7 +701,7 @@ ARE_THESE_REALLY_HARMFUL:
 
   // Cleanup.
   File::remove(file.str());
-  ignored = chdir(current_dir.c_str());
+  chdir(current_dir.c_str());
   return changes ? CmdEdit::editResult::changes : CmdEdit::editResult::nochanges;
 }
 

